webapp/lib/model.g.dart

```diff
@@ -58,11 +58,20 @@ class Conversation {
     return batch;
   }
 
-  DocBatchUpdate updateNotes(DocStorage docStorage, String documentPath, String newValue, [DocBatchUpdate batch]) {
-    notes = newValue;
-    batch ??= docStorage.batch();
-    batch.update(documentPath, data: {'notes': newValue});
-    return batch;
+  Future<bool> setNotes(DocPubSubUpdate pubSubClient, String newValue) {
```

Comment:
Consider calling this setNote or setConversationNote - the plurality here was making it a bit harder tor me to understand the difference between it and the set all code below

Reply:
@lukechurch The field is called `notes`, thus `set<name-of-field>`... should we rename the field? Or just this method?